src/App.php

```diff
@@ -203,7 +203,7 @@ public function processRequest(ServerRequestInterface $request, float $start_tim
 	/**
 	 * @internal
 	 */
-	public function processConsole(array $serverParams): void
+	public function processConsole(array $serverParams, bool $testmode = false): void
```

Comment:
What is the purpose of the `$testmode`? Can we move this information into `$serverParams`?

Reply:
I have some concerns with this change. The `App::processConsole()` is marked as internal because it will change in the near future. It should not be called by 3rd party code, even for custom dev scripts.

A better approach could be a new `FakeCommand` that will do nothing but can be called by your scripts: `bin/console.php dummy`